RELEASE_NOTES.md

```diff
@@ -1,4 +1,29 @@
-# ChefDK 3.12 Release Notes
+# ChefDK 3.12.10 Release Notes
+
+## Updated Components
+
+### Chef Infra Client 14.14.29
+
+Chef Infra Client has been updated to 14.14.29 with the following bug fixes:
+ - Fixed service/systemd_unit resource trying to reenable services with an indirect status
+ - systemd_unit now logs at info level
+ - Fixed knife config get throwing a TypeError: no implicit conversion of nil into String error
+
+### kitchen-digitalocean 0.10.4
+
+kitchen-digitalocean has been updated to 0.10.5, which adds new image aliases for Debian-10 and FreeBSD-12.
+
+### kitchen-dokkken 2.8.0
+
+kitchen-dokken has been updated to 2.8.0, which now exposes the `CI` and `TEST_KITCHEN` environmental variables to match the behavior of `kitchen-vagrant`.
```

Comment:
```suggestion
kitchen-dokken has been updated to 2.8.0. This will make the `CI` and `TEST_KITCHEN` environmental variables match the behavior of `kitchen-vagrant`.
```